pkg/config/config.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+type DB struct {
+	Host               string `yaml:"host"`
+	Port               int    `yaml:"port"`
+	User               string `yaml:"user"`
+	Password           string `yaml:"password"`
+	Name               string `yaml:"name"`
+	ProvidersTableName string `yaml:"providersTableName"`
+}
```

Comment:
Why is that a separate structure and not a nested struct as part of the config struct as requested in the last review?

Reply:
kept it as nested struct